app/views/home/small_bets.html.erb

```diff
@@ -0,0 +1,352 @@
+        name: "Shivam Dewan",
+        description: "Small Bets is my new idea generator. If you need inspiration and a supportive community, this is the place to be.",
+        image_url: image_url("smallbets/shivamdewan.png"),
+        bg_color: "bg-purple"
+      %>
+    </div>
+  </div>
+</div>
+
+<div class="flex flex-col border-t border-black overflow-hidden lg:flex-row">
+  <div class="bg-white py-16 px-8 flex items-center justify-center sm:p-12 md:p-16 lg:w-1/2 xl:p-24">
+    <div class="max-w-2xl space-y-12 md:space-y-8">
+      <div class="space-y-4">
+        <h3 class="font-medium text-3xl lg:text-4xl xl:text-5xl">
+          Video library of 50 classes
+        </h3>
+        <p class="text-lg lg:text-xl xl:text-2xl">
+          90 hours of high quality education on all topics in one place to level up your knowledge.
+        </p>
+      </div>
+      <ul class="grid grid-cols-1 gap-3 pl-0 text-lg md:grid-cols-2">
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          Make $500 on Upwork by Monday - Sean O'Dowd
+        </li>
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          Emerging from the void on X - Daniel Vassallo
+        </li>
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          Growing on Substack - Elle Griffin
+        </li>
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          AI for the rest of us - Sairam Sundaresan
+        </li>
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          Intro to SEO - Jordan O'Connor
+        </li>
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          Bulding & launching macOS apps with SwiftUI - Grace Huang
+        </li>
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          A practical guide to self-employment taxes - Catherine Cusick
+        </li>
+        <li class="flex items-center">
+          <span class="mr-3 flex items-center justify-center w-8 h-8">
+            <%= icon "solid-check-circle" %>
+          </span>
+          Crowdfunding crash course - Tugra Sahiner
+        </li>
+        <li class="flex items-center">
+          +42 more!
```

Comment:
At some point I'd love to make this expand to show the full list, but I'm happy to go live with it like this and figure out the design for that later.